core/templates/dev/head/pages/exploration-player-page/templates/flag-exploration-modal.template.html

```diff
@@ -48,4 +48,12 @@ <h4>
     margin-bottom: 20px;
     margin-top: 20px;
   }
+  .oppia-radio-button input {
+  height: 15px;
```

Comment:
Indent this and the next line by 2. Ditto below.

Also, the name is too general. If it's only used here, call it oppia-flag-modal-radio-button or similar. Ditto above: oppia-flag-modal-long-text. Again, this is only if the class is being used here.

Reply:
Done it was only used here Thanks!